MAUI/Cartesian-Charts/neumorphicUI-column-chart.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: post
+title: Neumorphic UI Column Chart in .NET MAUI Chart control | Syncfusion
+description: Learn here about how to create the Neumorphic UI chart types using Syncfusion .NET MAUI Chart (SfCartesianChart) control.
```

Comment:
@DhanarajRajendran 
In description, mention as -> Neumorphic UI column chart

Reply:
Addressed the comments.